src/main/java/Duke.java

```diff
@@ -1,10 +1,46 @@
+import duke.task.TaskException;
+import duke.task.TaskList;
+import duke.util.FileManager;
+import duke.util.Parser;
+import duke.util.Ui;
+
+import java.util.*;
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    private String filePath = "./data/saveData.txt";
+    private FileManager storage;  // This is the storage class just named duke.util.FileManager
+    private TaskList tasks;
+    private Ui ui;
+    private Parser parser;
+
+    public Duke(String filePath) {
```

Comment:
I like your OOP abstraction ideas which makes your main class look clean and easy to understand. LGTM!